README.md

```diff
@@ -92,7 +92,7 @@ function App() {
     // perform some logic based on the visitor data
     return (
       <div>
-        Welcome {data.visitorFound ? 'back' : ''}!
+        Welcome {data.visitorFound ? `back ${data.visitorId}` : ''}!
```

Comment:
@makma I didn't put it here deliberately because the visitorId shouldn't be displayed anywhere, it's part of the website security, so it's not a good practice ti show it to the user. Why do you think this change is necessary?

Reply:
@borislobanov92 
- I thought it was a typo and you wanted to demonstrate logic using `visitorId` (I assumed it, because of the comment - `// perform some logic based on the visitor data`)
- There's not a single mention about visitorId in the readme. I assume in the majority of cases, users will send `visitorId` (or `requestId`) within their payload to their backend.

Reply:
@makma I think we can log `visitorId` to the console but not display it to the user. We also might want to mention it in readme but I don't think that the change that you're proposing (displaying `visitorIs` in the example) should be merged. It wasn't typo from my part and the comment implied that the user is free to do whatever they want with it. 
Can we close this PR or do you still think we need this change?

Reply:
Sure, sounds good, thanks @borislobanov92